### PR TITLE
New icons for resource types

### PR DESCRIPTION
**Pull request by elizavetaRa**
## Description
- drawio
- ifc
- ipynb

### CODE REVIEW

**Comment by elizavetaRa** [comment]
@tbsbdr @pascalwengerter @kulmann Guys, ping me if there are any other icons that are needed. We also have a jsroot icon, I don't know if it is needed upstream. And ping if some properties inside svg's need to be set in some other way.

**Comment by diocas** [comment]
@elizavetaRa jupyter? def is more common than ifc

**Review by pascalwengerter — CHANGES_REQUESTED** [review]
Feature branch would need a review, merge & rebase with changes in https://github.com/cernbox/owncloud-design-system/pull/222 included. Then I'm happy to proceed very quickly here :)

File: src/assets/icons/resource-type-jupyter-fill.svg [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      />
+   </g>
+   <path
+      id="path19604"
+      fill="none"
+      d="m 205.98235,394.67457 c -43.61728,0 -82.03308,-10.23461 -101.85458,-33.3518 7.69047,20.79647 21.56844,38.73789 39.76407,51.40896 18.19563,12.67167 39.83697,19.46397 62.0099,19.46397 22.17325,0 43.81398,-6.7923 62.01021,-19.46397 18.19563,-12.67107 32.073,-30.61249 39.76437,-51.40896 -19.7733,23.11719 -57.88271,33.3518 -101.69397,33.3518 z"
+      style="mix-blend-mode:normal;fill:#f37726;stroke-width:5.44357"
```
Comment (pascalwengerter) [inline src/assets/icons/resource-type-jupyter-fill.svg:13]:
```suggestion
      style="mix-blend-mode:normal;stroke-width:5.44357"
```

File: src/assets/icons/resource-type-jupyter-fill.svg [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      />
+   <path
+      id="path19601"
+      d="m 205.90174,254.5979 c 43.61759,0 81.95279,10.82484 101.77458,33.94203 -7.69137,-20.79647 -21.56874,-38.73789 -39.76437,-51.40896 -18.19623,-12.67106 -39.83696,-19.46397 -62.01021,-19.46397 -22.17293,0 -43.81427,6.79291 -62.0099,19.46397 -18.19563,12.67107 -32.0736,30.61249 -39.76407,51.40896 19.8215,-23.16538 57.96301,-33.94203 101.77397,-33.94203 z"
+      style="mix-blend-mode:normal;fill:#f37726;stroke-width:5.44357"
```
Comment (pascalwengerter) [inline src/assets/icons/resource-type-jupyter-fill.svg:18]:
```suggestion
      style="mix-blend-mode:normal;stroke-width:5.44357"
```

**Review by pascalwengerter — CHANGES_REQUESTED** [review]
Please omit the colors from the SVG, they will later be replaced by `var(--oc-color-icon-ifc)` according to the mapping we implemented (and make sure the color you desire is the right one in the color.yaml file)

**Pull request merged**